pkg/kubectl/cmd/apply.go

```diff
@@ -123,6 +128,27 @@ func validatePruneAll(prune, all bool, selector string) error {
 	return nil
 }
 
+func validatePruneResources(pruneResources *[]pruneResource, gvks []string) error {
```

Comment:
It's weird that this function is called `validatePruneResources` but it's actually modifying it's input argument. Is it doing to much or should it be renamed?